src/hash_table.py

```diff
@@ -0,0 +1,37 @@
+"""Hash Table implementation."""
+
+
+class HashTable(object):
+    """Class HashTable."""
+
+    def __init__(self, num):
+        """Initialize with number of slots to be in table."""
+        self.table = []
+        for x in range(0, num):
```

Comment:
range defaults to having 0 as the first argument.
This works fine, but you could also just say range(num).